CHANGELOG.rst

```diff
@@ -2,19 +2,28 @@
 Changelog
 =========
 
-2.1.0-rc.0 (2023-03-03)
-=======================
+2.1.0 "Gers" (2023-03-14)
+=========================
 
-We are proud to announce a release candidate of Open Forms 2.1!
+.. epigraph::
 
-This release candidate has focused on stability issues compared to the previous alpha
-version and includes some new experimental features.
+   **Gers** *[Gers]• Gaaf/mooi/leuk/geweldig/tof/heel goed*
+
+   -- Rotterdams Woordenboek
+
+Upgrade procedure
+-----------------
+
+Ensure that your current version of Open Forms is at least version 2.0.2 before
```

Comment:
Doublecheck: Are we blocking upgrades from before 2.0.2 to 2.1.0?

Reply:
yes, 2.0.0 or 2.0.1 to 2.1.0 is not possible, see https://github.com/open-formulieren/open-forms/blob/master/src/openforms/upgrades/upgrade_paths.py#L84